packages/vscode-extension/lib/wrapper.js

```diff
@@ -407,6 +410,28 @@ export function AppWrapper({ children, initialProps, fabric }) {
     }
   }, [!!devtoolsAgent && hasLayout]);
 
+  useEffect(() => {
+    if (!devtoolsAgent) {
+      return;
+    }
+    InternalImports.updateInstrumentationOptions({
+      reportRenders: (blueprintOutlines) => {
+        devtoolsAgent._bridge.send("RNIDE_rendersReported", { blueprintOutlines });
+      },
+    });
+    devtoolsAgent._bridge.send("RNIDE_queryInstrumentationOptions");
```

Comment:
I don't think this call is necessary. This seems to follow similar scheme as the network plugin. In that case we wait for devtools to connect before we send enable command. This way we can be sure that we always get the enabled state event even when it was sent before reload occured.

Reply:
fixed by emitting `RNIDE_updateInstrumentationOptions` on `RNIDE_appReady` (extension side)